samples/snippets/src/test/java/com/example/bigquerystorage/WriteCommittedStreamIT.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquerystorage;
+
+import static com.google.common.truth.Truth.assertThat;
+import static junit.framework.TestCase.assertNotNull;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class WriteCommittedStreamIT {
+
+  private static final String GOOGLE_CLOUD_PROJECT = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String BIGQUERY_DATASET_NAME = System.getenv("BIGQUERY_DATASET_NAME");
+  private static final String BIGQUERY_TABLE_NAME = System.getenv("BIGQUERY_TABLE_NAME");
```

Comment:
i think we should just create a new table and destroy this table each time when we do testing so that we don't flood our project w these random records.

Here is an example: https://github.com/googleapis/java-bigquery/blob/master/samples/snippets/src/test/java/com/example/bigquery/CopyTableCmekIT.java#L63-L90

Reply:
Secrets are set up in this PR https://github.com/googleapis/java-bigquerystorage/pull/776. You should be good to go! 

Tag `kokoro:force-run` label to re-trigger integration tests.

Reply:
Thanks! I added code to create and delete some temporary datasets and tables in the tests. I don't think I have permission to add a label.